From bc5cf83e2756407b5a65a53258d690ea015e8544 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Tue, 26 Sep 2023 10:51:56 +0200
Subject: [PATCH 0553/1141] clk: stm32mp25: fix parent of clk_lanebyte clock

Modify parent of clk_lanebyte (ck_ker_ltdc instead
ck_flexgen_27)
We have to take account the gate of ck_ker_ltdc clock.

Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: I6f8b06ab9845d51b81c3fe3dc40674bb443053a8
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/329759
Domain-Review: Yannick FERTRE <yannick.fertre@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 drivers/clk/stm32/clk-stm32mp25.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/clk/stm32/clk-stm32mp25.c
+++ b/drivers/clk/stm32/clk-stm32mp25.c
@@ -77,7 +77,7 @@ static const char * const usb3pciphy_src
 };
 
 static const char * const dsiblane_src[] = {
-	"clk_phy_dsi", "ck_flexgen_27"
+	"clk_phy_dsi", "ck_ker_ltdc"
 };
 
 static const char * const dsiphy_src[] = {
